contracts/mock/MockRewardsDistributor.sol

```diff
@@ -17,23 +17,29 @@ contract MockRewardsDistributor is IRewardsDistributorAdmin, Ownable {
     uint256 public compSupplySpeed;
     uint256 public compBorrowSpeed;
 
+    address public pendingNewAdmin;
+    address public newAdmin;
+
     constructor() Ownable() {}
 
     /**
       * @notice Begins transfer of admin rights. The newPendingAdmin must call `_acceptAdmin` to finalize the transfer.
       * @dev Admin function to begin change of admin. The newPendingAdmin must call `_acceptAdmin` to finalize the transfer.
-      * @param newPendingAdmin New pending admin.
+      * @param _newPendingAdmin New pending admin.
       */
-    function _setPendingAdmin(address newPendingAdmin) external override onlyOwner {
-        emit SuccessSetAdmin(true);
+    function _setPendingAdmin(address _newPendingAdmin) external override onlyOwner {
```

Comment:
Setters in our system by convention emit both the old and new one, see https://github.com/fei-protocol/fei-protocol-core/blob/master/contracts/bondingcurve/BondingCurve.sol#L259-L265 match the code style going forward

Reply:
including capitalization, event names, etc

Reply:
Events should not be camelCase.